fix(py,ts): support OME-Zarr RFC-3 (arbitrary axis dimensions, names, types, ordering) (#611)
thewtex/ngff-zarr: docs/python.md

>>> multiscales = nz.from_ngff_zarr('cthead1.ome.zarr')
```

> OME-Zarr version 0.1 to 0.6 is supported. Version 0.6 adds RFC-5 coordinate systems and transformations. The opt-in development version `0.9.dev1` additionally adopts RFC-3, lifting every restriction on axis count, names, types and order.

Reviewer: ```suggestion
OME-Zarr version 0.1 to 0.6 is supported. Version 0.6 adds RFC-5 coordinate systems and transformations. The opt-in development version `0.9.dev1` additionally adopts RFC-3, that expands supported axis counts, names, types and order.
```
